router/80_to_443_redirect.conf

```diff
@@ -0,0 +1,8 @@
+; server {
+;     listen 80;
+;     server_tokens off;
+;
+;     location / {
+;         return 301 https://$host$request_uri;
```

Comment:
I think that for now our application should be able to accept both `80` and `443` requests. We can then redirect accordingly to some endpoints like `/secure/...`, `/unsecure/..`. What do you think?

Reply:
Or is it just to redirect `a.com` and `b.com` to 443? If so then forget about upper comment.

Reply:
This will redirect all traffic from 80 to 443 but for now we also want to accept requests to port 80 so now I commented it.